lib/dictBuilder/zdict.h

```diff
@@ -19,13 +19,13 @@ extern "C" {
 #include <stddef.h>  /* size_t */
 
 
-/*======  Export for Windows  ======*/
-/*!
-*  ZSTD_DLL_EXPORT :
-*  Enable exporting of functions when building a Windows DLL
-*/
-#if defined(_WIN32) && defined(ZSTD_DLL_EXPORT) && (ZSTD_DLL_EXPORT==1)
+/* =====   ZDICTLIB_API : control library symbols visibility   ===== */
+#if defined(__GNUC__) && (__GNUC__ >= 4)
```

Comment:
This breaks DLL compiled with gcc/MinGW. A fix for zstd.h is available here:
https://github.com/inikep/zstd/commit/60f10aab6c26dda2224583de84fe64ab7a649f55